src/components/FormAlert.js

```diff
@@ -0,0 +1,32 @@
+import React from 'react';
+import { FirestoreCollection } from 'react-firestore';
+
+function normalizeItem(item) {
+  var pattern = /\d[.,\/#!$%\^&\*;:{}=\-_`~()]\ix/;
+  // remove capitilization
+  item.replace(pattern, '');
+  return;
+  // remove numbers and special characters
+  // spaces
+  //return item normalized
+}
+
+function FormAlert(props) {
+  return (
+    <FirestoreCollection
+      path="items"
+      filter={['name', '==', props.userSubmit]}
+      render={({ data }) => {
+        if (data.length > 0) {
+          return (
+            <p>This item has already been added to your shopping list.</p> // if we can add the actual item name {data.name}
+          );
+        } else {
+          return <div></div>; //Is there a better way to have an empty return
```

Comment:
You can always use a short circuit pattern for a `render this or nothing`:

```jsx
return someCondition && <p>Hey, the condition was true!</p>;
```

Reply:
I just now figured out how to use @stevelikesmusic's suggestion in this context:

```
          render={({ data }) => {
            return data.length > 0 ? (
              <p>
                {data[0].name} has already been added to your shopping list.
              </p>
            ) : null;
          }}
```